Dockerfile

```diff
@@ -20,11 +20,13 @@ RUN set -x \
     zlib1g-dev \
     libfreetype6-dev \
     python-virtualenv \
+    apt-utils \
```

Comment:
Do we really need this? I thought it is just for the speed-up of the installation process with `apt`.  Of course this is nothing bad, but it would expand the docker image, and I would prefer to keep the image as small as possible... In addition the advantage of the apt-utils package we have after installing it, but we have no second `apt-get install`.

I would remove this line / package, or if there is a sense maybe purge it in a last command?

What is your opinion?

Reply:
I just copy/pasted some stuff from the web; I doubt we need.  We should probably remove `apt-utils`

Reply:
OK, then I will change it after merge and make version-branches for the releases